spec/elastic_apm/context/request/socket_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module ElasticAPM
+  class Context
+    class Request
+      RSpec.describe Socket do
+        subject { described_class.new req }
+
+        context 'with an ip' do
+          let(:req) do
+            Rack::Request.new(
+              Rack::MockRequest.env_for('/', 'REMOTE_ADDR' => '127.0.0.1')
+            )
+          end
+
+          its(:remote_addr) { is_expected.to eq '127.0.0.1' }
+        end
+
+        # 'Trusted' as per Rack's definition:
+        # https://github.com/rack/rack/blob/2-0-stable/lib/rack/request.rb#L419-L421
+        context 'with a "trusted" remote addr and forwarding header' do
```

Comment:
This is really minor, but it might be better to link to a tagged version's file. If you link to the lines in a branch, they may change as time goes on and we lose the reference.
For example https://github.com/rack/rack/blob/2.0.7/lib/rack/request.rb#L419-L421